app/controller/sdl/Abstract/Controller.js

```diff
@@ -1549,5 +1549,26 @@ SDL.SDLController = Em.Object.extend(
       }
       return title;
     },
+
+    /**
+     * @description Callback for display image mode change.
+     */
+    imageModeChanged: function() { 
+      if (!SDL.SDLController.model) {
+        return;
+      }
+
+      SDL.SDLController.model.setMode(SDL.SDLModel.data.imageMode);
+      var commands = SDL.SDLController.model.get('currentCommandsList');
+      const length = SDL.OptionsView.commands.items.length;
+      for(var i=0;i<length;i++){
```

Comment:
@YarikMamykin I believe using of `forEach` would be simpler here

Reply:
@AKalinich-Luxoft nope, because, as you can see, we iterate through `SDL.SDLController.model.currentCommandsList` and `SDL.OptionsView.commands.items` at one time.